presto-main/src/main/java/com/facebook/presto/connector/system/SystemSplitManager.java

```diff
@@ -74,7 +75,7 @@ public ConnectorSplitSource getSplits(ConnectorSession session, ConnectorTableLa
             nodes.addAll(nodeManager.getCoordinators());
         }
         else if (tableDistributionMode == ALL_NODES) {
-            nodes.addAll(nodeManager.getActiveNodes());
+            nodes.addAll(nodeManager.getNodes(ACTIVE));
```

Comment:
I think this also needs to include shutting down nodes. Otherwise you won't be able to query the tasks system table on those nodes